differs/apt_diff.go

```diff
@@ -120,3 +123,61 @@ func parseLine(text string, currPackage string, packages map[string]util.Package
 	}
 	return currPackage
 }
+
+type AptLayerAnalyzer struct {
+}
+
+func (a AptLayerAnalyzer) Name() string {
+	return "AptLayerAnalyzer"
+}
+
+// AptDiff compares the packages installed by apt-get.
+func (a AptLayerAnalyzer) Diff(image1, image2 pkgutil.Image) (util.Result, error) {
+	diff, err := singleVersionLayerDiff(image1, image2, a)
+	return diff, err
+}
+
+func (a AptLayerAnalyzer) Analyze(image pkgutil.Image) (util.Result, error) {
+	analysis, err := singleVersionLayerAnalysis(image, a)
+	return analysis, err
+}
+
+func (a AptLayerAnalyzer) getPackages(image pkgutil.Image) ([]map[string]util.PackageInfo, error) {
```

Comment:
Looks like this shares a lot of code with `AptAnalyzer.getPackages()` in `apt_diff.go`. Could you pull this out into a shared method between the two implementations?

Reply:
Sure you are right.